06week/checkpoint2.js

```diff
@@ -0,0 +1,103 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+let nums = [];
+const strAsNum = strNums.map(str => nums.push(str));
+console.log(strAsNum);
+// Find the sum of the even values
+let sumEvens = [];
+const evenSum = strNums.reduce((total, currentValue) => currentValue % 2 === 0 ? sumEvens.push(total + currentValue) : total, 0)  
```

Comment:
This is so close.  The reduce function that is being called is returning the length of the array after it's executed.  Just a quick syntax tweak would correct.  The expected outcome is 1950